internal/controller/config/resource_class.go

```diff
@@ -0,0 +1,60 @@
+package config
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	CommandContainerName = "container-0"
+)
+
+type ResourceClass struct {
```

Comment:
Please add doc comments to exported identifiers

```suggestion
// ResourceClass represents a reusable resource configuration.
type ResourceClass struct {
```